admin_guide/managing_networking.adoc

```diff
@@ -1186,9 +1186,12 @@ $ oc patch hostsubnet node1 -p \
 [IMPORTANT]
 ====
 Egress IPs are implemented as additional IP addresses on the primary network
-interface, and must be in the same subnet as the node's primary IP. Allowing
-additional IP addresses on the primary network interface might require extra
-configuration when using some cloud or VM solutions.
+interface, and must be in the same subnet as the node's primary IP.
+Additionally, any external IPs should not be configured in any Linux network
+configuration files, such as, `ifcfg-eth0`.
```

Comment:
Since this is a file name, should it be formatted as the following?

     *_ifcfg-eth0_* 

Also, perhaps no , after such as?

Reply:
Oh right. Hazards of after-edits... thanks @ahardin-rh